bitsail-test/bitsail-test-end-to-end/bitsail-test-e2e-base/src/main/java/com/bytedance/bitsail/test/e2e/executor/generic/GenericExecutor.java

```diff
@@ -81,7 +83,7 @@ public void init() {
             DockerLoggerFactory.getLogger(setting.getExecutorImage())).withSeparateOutputStreams())
         .withStartupAttempts(1)
         .withWorkingDirectory(EXECUTOR_ROOT_DIR.toAbsolutePath().toString())
-        .withCommand("bash", "-c", String.join(" ;", initCommands))
```

Comment:
Excuse me, but why change the `bash` to `sh`?

Reply:
The container image **testcontainers/ryuk** for testing does not have bash shell, it originally contain ash. If we do not change, to run test function `testCreateGenericExecutor()` would raise error below. For compatibility with different default shell, so I change `bash` to `sh`.
```
java.util.concurrent.CompletionException: org.testcontainers.containers.ContainerLaunchException: Container startup failed

  at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:273)
  at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:280)
  at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:722)
  at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:701)
  at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:456)
  at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
  at java.lang.Thread.run(Thread.java:750)
Caused by: org.testcontainers.containers.ContainerLaunchException: Container startup failed
  at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:349)
  at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:322)
  at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:719)
  ... 5 more
Caused by: org.rnorth.ducttape.RetryCountExceededException: Retry limit hit with exception
  at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:88)
  at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:334)
  ... 7 more
Caused by: org.testcontainers.containers.ContainerLaunchException: Could not create/start container
  at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:542)
  at org.testcontainers.containers.GenericContainer.lambda$doStart$0(GenericContainer.java:344)
  at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:81)
  ... 8 more
Caused by: com.github.dockerjava.api.exception.BadRequestException: Status 400: {"message":"failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: \"bash\": executable file not found in $PATH: unknown"}
```

Reply:
Okay.